# Generated by Django 5.0.6 on 2024-06-20 20:05

import Usuarios.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('Usuarios', '0019_alter_usuario_cpf_alter_usuario_email'),
    ]

    operations = [
        migrations.AlterField(
            model_name='usuario',
            name='cpf',
            field=models.CharField(error_messages={'max_length': 'cpf inválido, digite um cpf que tenha no maximo %(model.max_length) caracteres', 'unique': 'Já existe um usuário com esse cpf'}, max_length=14, unique=True, validators=[Usuarios.validators.validate_cpf]),
        ),
        migrations.AlterField(
            model_name='usuario',
            name='dataNascimento',
            field=models.DateField(validators=[Usuarios.validators.validate_date]),
        ),
    ]
